M/MPFI/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MPFI"
+version = v"1.5.4"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://gforge.inria.fr/frs/download.php/file/38111/mpfi-1.5.4.tgz", "3b3938595d720af17973deaf727cfc0dd41c8b16c20adc103a970f4a43ae3a56"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+    cd $WORKSPACE/srcdir/mpfi-*
+
+    # Remove the doc targets from the build. We don't need them, and they require
+    # the texinfo program to build properly
+    atomic_patch ${WORKSPACE}/srcdir/patches/remove-doc-build.patch
+
+    # The autotools scripts are not provided setup, so we must configure the configure script
+    ./autogen.sh
+
+    ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --with-gmp=${prefix} --with-mpfr=${prefix}
```

Comment:
Let's not build the static archive, which will also make the tarball much smaller:
```suggestion
    ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --with-gmp=${prefix} --with-mpfr=${prefix} --disable-static
```

Reply:
These tarballs are only 300k already... so they are already pretty small.

Reply:
But we don't use the static archives anyway, they just drain storage on the user's machine.  We routinely get issues because of them: #642, #668, #840, #841, #864.